alerts/cluster-monitoring-operator/PrometheusDuplicateTimestamps.md

```diff
@@ -0,0 +1,83 @@
+# PrometheusDuplicateTimestamps
+
+## Meaning
+
+The `PrometheusDuplicateTimestamps` alert is triggered when there is a constant
+increase in dropped samples due to them having identical timestamps.
+
+## Impact
+
+Unwanted samples might be scraped, and desired samples might be dropped.
+Consequently, queries may yield incorrect and unexpected results.
+
+## Diagnosis
+
+1. Determine whether the alert has triggered for the instance of Prometheus used
+   for default cluster monitoring or for the instance that monitors user-defined
+   projects by viewing the alert message's `namespace` label: the namespace for
+   default cluster monitoring is `openshift-monitoring`; the namespace for user
+   workload monitoring is `openshift-user-workload-monitoring`.
+
+2. Review the logs for the affected Prometheus instance:
+
+   ```shell
+   $ NAMESPACE='<value of namespace label from alert>'
+
+   $ oc -n $NAMESPACE logs -l 'app.kubernetes.io/name=prometheus' | \
+   grep 'Error on ingesting samples with different value but same timestamp.*' \
+   | sort | uniq -c | sort -n
+   level=warn ... scrape_pool="the-scrape-pool" target="an-involved-target" \
+   msg="Error on ingesting samples with different value but same timestamp"
+   ```
+   Warning logs similar to the one above should be present.
+
+## Mitigation
+
+If the alert originates from the `openshift-monitoring` namespace, please open a
+support case. If not, proceed as follows:
+
+The alert may fire if a target is duplicated: defined multiple times with
+identical target labels. You can learn if the logs above reveal that the same
+target is defined in different scrape pools.
+
+Also, this may occur if distinct targets across different scrape pools are
+producing the same samples.
+
+In these two cases, the logs above should be sufficient to guide you to the
+right place in the configuration file to find and fix the conflicting targets:
+remove the duplicated targets or ensure that distinct targets are labeled
+uniquely.
```

Comment:
IMHO we should rather explain how to map from the scrape pool to the ServiceMonitor/PodMonitor faulty objects.

Reply:
Done, tell me if I should be even more explicit.